include/zephyr/net/net_pkt.h

```diff
@@ -2476,6 +2471,18 @@ static inline int net_pkt_write_le16(struct net_pkt *pkt, uint16_t data)
  */
 size_t net_pkt_remaining_data(struct net_pkt *pkt);
 
+/**
+ * @brief Get the total amount of bytes stored in the packet.
```

Comment:
Nit: 

```suggestion
 * @brief Get the total amount of bytes stored in a packet.
```

Reply:
Hmm, I am not native English speaker so I am probably way off but using "the" here could point to the exact pkt parameter we have here.

Reply:
"the" is usually used in the `@param` text, while the function doc would usually use "a/an" as it describes what the function does in a very general way. Also just a matter of consistency in this file really, just look at docs for `net_pkt_update_length` (just next in the file), `net_pkt_frag_del`, `net_pkt_memset`, etc :)

Reply:
Updated